thirdeye-ui/src/app/components/home-page/recent-anomalies/recent-anomalies.component.tsx

```diff
@@ -108,29 +110,43 @@ export const RecentAnomalies: FunctionComponent = () => {
                         </>
                     }
                 >
-                    <Table size="small">
-                        <TableHead>
-                            <TableRow>
-                                <TableCell>Anomaly ID</TableCell>
-                                <TableCell>Alert Name</TableCell>
-                                <TableCell>Metric</TableCell>
-                                <TableCell>Started</TableCell>
-                                <TableCell>Ended</TableCell>
-                                <TableCell>Deviation</TableCell>
-                                <TableCell />
-                                <TableCell />
-                            </TableRow>
-                        </TableHead>
-                        {anomaliesToDisplay.length > 0 &&
-                            anomaliesToDisplay.map((anomaly) => {
+                    {anomaliesToDisplay.length > 0 ? (
+                        <Table size="small">
+                            <TableHead>
+                                <TableRow>
+                                    <TableCell>Anomaly ID</TableCell>
+                                    <TableCell>Alert Name</TableCell>
+                                    <TableCell>Metric</TableCell>
+                                    <TableCell>Started</TableCell>
+                                    <TableCell>Ended</TableCell>
+                                    <TableCell>Deviation</TableCell>
+                                    <TableCell />
+                                    <TableCell />
+                                </TableRow>
+                            </TableHead>
+                            {anomaliesToDisplay.map((anomaly) => {
                                 return (
                                     <AnomalyRow
                                         anomaly={anomaly}
                                         key={anomaly.id}
                                     />
                                 );
                             })}
-                    </Table>
+                        </Table>
+                    ) : (
+                        <Box
+                            alignItems="center"
+                            justifyContent="center"
+                            mb={8}
+                            mt={8}
+                            textAlign="center"
+                            width="100%"
+                        >
+                            <NoDataIndicator>
+                                No recent anomalies found in the last 6 months.
+                            </NoDataIndicator>
+                        </Box>
+                    )}
```

Comment:
Let's use https://github.com/startreedata/thirdeye/blob/master/thirdeye-ui/src/app/components/page-states/empty-state-switch/empty-state-switch.component.tsx

```
<EmptyStateSwitch
    emptyState={
        <Box
            alignItems="center"
            justifyContent="center"
            pb={8}
            pt={8}
            textAlign="center"
            width="100%"
        >
            <NoDataIndicator>
                No recent anomalies found in the last 6 months.
            </NoDataIndicator>
        </Box>
    }
    isEmpty={anomaliesToDisplay.length === 0}
>
    // Main content
</EmptyStateSwitch>
```

When adding empty space, I use padding instead of margin

Reply:
Implemented accordingly